integration/findandmodify_test.go

```diff
@@ -152,34 +152,6 @@ func TestFindAndModifySimple(t *testing.T) {
 	}
 }
 
-func TestFindAndModifyEmptyCollectionName(t *testing.T) {
```

Comment:
Why is this test deleted? Empty DB name and empty collection name look like totally different cases two me and they will fail because of different reasons - `empty DB name + valid collection name` vs `valid DB name + empty collection name`

Reply:
> Why is this test deleted?

I've added the description in the PR before requesting review..
 
> Empty DB name and empty collection name look like totally different cases two me and they will fail because of different reasons - `empty DB name + valid collection name` vs `valid DB name + empty collection name`

Yes it is and `CollectionNameTest` does this test.

Reply:
> > removed `TestFindAndModifyEmptyCollectionName` test as the same test already done in `TestCollectionName`.
> 
> `TestCollectionName` tries to create collection with an empty name. This test tries to find something in non-existing collection without creating it. I don't understand how they are the same

That's an argument, restored.